test/integration/admin/api/api_docs_services_controller_test.rb

```diff
@@ -10,13 +10,13 @@ def setup
 
   class MasterAccountTest < Admin::Api::ApiDocsServicesControllerTest
     def test_index_json_saas
-      get admin_api_docs_services_path
+      get admin_api_active_docs_path
```

Comment:
It was mistakenly testing the same as this:
https://github.com/3scale/porta/blob/340cff144aea10d2cca9ccd58909b49d7cecbe6a/test/integration/admin/api_docs/services_controller_test.rb#L40-L41